package.json

```diff
@@ -172,6 +172,7 @@
   "homepage": "https://github.com/mozillla/addons-frontend#readme",
   "dependencies": {
     "babel-polyfill": "6.26.0",
+    "babel-runtime": "^6.26.0",
```

Comment:
hmm..I guess it does fix as devDep too :)  I guess I thought this might be an issue when we need to run on gh-pages tho...?

I'll update for now and circle back..

Reply:
I don't think so, `gh-pages` host static content anyway.

Reply:
ahh yep.. i think you're right, thanks